### PR TITLE
update mysql compose image and add gradle compose plugin

### PR DESCRIPTION
**Pull request by Danilosimoes**
update mysql docker compose image and add gradle compose plugin

### CODE REVIEW

**Review by robsonoduarte — COMMENTED** [review]
@Danilosimoes where is the gradle compose plugin configuration :eyes:

File: docker-compose.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 version: '3.3'
 services:
   mysql:
-    image: mysql:5.7.38
+    image: mysql:8.0
```
Comment (robsonoduarte) [inline docker-compose.yml:4]: the last version of mysql docker image is 8.0.33:
https://hub.docker.com/_/mysql/tags

**Comment by robsonoduarte** [comment]
@Danilosimoes this is the plugin:
https://github.com/avast/gradle-docker-compose-plugin

the quick star is all we need now:
image

File: docker-compose.yml (hunk 1 above)
Comment (Danilosimoes) [reply docker-compose.yml]: I change the mysql docker image to 8.0.33 and is working

Comment (Danilosimoes) [reply docker-compose.yml]: And I add the gradle plugin configuration, I analyzed the code then i´d need add the buildscrpit block, beacuse we had yhe configurantion in plugun, like is wrote in quick stat, then i just added the apply docker compose.
Its ok or it not ok?

Comment (Danilosimoes) [reply docker-compose.yml]: I added the buildscript block!

**Comment by robsonoduarte** [comment]
@Danilosimoes I send a patch...

1 - using the [plugin dsl](https://docs.gradle.org/current/userguide/plugins.html#sec:plugins_block) 
2 - create a new application-test to use the mysql local that was up by docker compose
3 - configure the plugin to up and down the docker continers during in gradle test fase

**Review by robsonoduarte — APPROVED** [review]
sounds good :muscle:

**Pull request merged**